tests/fiaas_deploy_daemon/deployer/kubernetes/test_role_binding.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+from unittest.mock import create_autospec, patch, call
+from k8s.client import NotFound
+from k8s.models.role_binding import RoleBinding
+from fiaas_deploy_daemon.deployer.kubernetes.role_binding import RoleBindingDeployer
+from fiaas_deploy_daemon.deployer.kubernetes.owner_references import OwnerReferences
+from fiaas_deploy_daemon.config import Configuration
+
+
+class TestRoleBindingDeployer:
+
+    @pytest.fixture
+    def owner_references(self):
+        return create_autospec(OwnerReferences, spec_set=True, instance=True)
+
+    @pytest.fixture
+    def deployer(self, owner_references):
+        config = create_autospec(Configuration([]), spec_set=True)
+        config.list_of_roles = ["test-role-1", "test-role-2"]
+        config.list_of_cluster_roles = ["cluster-role-1", "cluster-role-2"]
+        return RoleBindingDeployer(config, owner_references)
+
+    def test_create_bindings(self, deployer, owner_references, app_spec):
+        def _test_create_bindings(roles_list, role_kind):
+            with patch.object(RoleBinding, 'get') as mock_get, \
+                 patch.object(RoleBinding, 'save') as mock_save:
+
+                mock_get.side_effect = NotFound
+                deployer._create_role_bindings(app_spec, roles_list, role_kind, 1)
+
+                mock_get.assert_has_calls([
+                    call(f"{app_spec.name}-1", app_spec.namespace),
+                    call(f"{app_spec.name}-2", app_spec.namespace)
+                ])
+                mock_save.assert_has_calls([
+                    call(),
+                    call()
+                ])
+                owner_references.apply.assert_called()
+
+        _test_create_bindings(["test-role-1", "test-role-2"], "Role")
+        _test_create_bindings(["cluster-role-1", "cluster-role-2"], "ClusterRole")
```

Comment:
Can you try to use a parametrize here?
See https://github.com/fiaas/fiaas-deploy-daemon/blob/master/tests/fiaas_deploy_daemon/deployer/kubernetes/test_ready_check.py#L128-L137 as example

Reply:
Nice!!, I've parameterized it now. Take a look and let me know if you need any further changes